audio.py

```diff
@@ -88,3 +136,33 @@ def compute_duration(audio_path, backend = 'ffmpeg'):
 	cmd = ['ffprobe', '-v', 'error', '-show_entries', 'format=duration', '-of', 'default=noprint_wrappers=1:nokey=1'
 			] if backend == 'ffmpeg' else ['soxi', '-D'] if backend == 'sox' else None
 	return float(subprocess.check_output(cmd + [audio_path]))
+
+
+def timeit(audio_path, numbers, sample_rate, mono, audio_backend, delimiter):
+	start_process_time = time.process_time_ns()
+	start_perf_counter = time.perf_counter_ns()
+	for i in range(numbers):
+		signal, sample_rate = read_audio(audio_path, sample_rate = sample_rate, mono = mono, backend = audio_backend)
+	end_process_time = time.process_time_ns()
+	end_perf_counter = time.perf_counter_ns()
+	process_time = (end_process_time - start_process_time) / delimiter
+	perf_counter = (end_perf_counter - start_perf_counter) / delimiter
+	print(audio_path, numbers, audio_backend, process_time, perf_counter)
```

Comment:
Please do markdown-ready formatting instead, i.e. sperate the columns by "|"

Please round the results, sub-microsecond precision is unnecessary here. Please pad the columns to fixed width, as done in train.py. This would make the numbers aligned and readable directly in terminal - which is useful when introducing a new backend.

Reply:
done